scantools/scanners/navvis/camera_tiles.py

```diff
@@ -111,8 +111,15 @@ def _config(self, img_width, img_height, tile_format):
                 raise ValueError("Non-implemented tile format:", tile_format, "for device:", self.device)
 
         elif self.device == 'VLX':
+            # no tiles
+            if tile_format is TileFormat.TILES_none:
```

Comment:
How did we come up with this defaults?
Shouldn't we enforce always tiling?

Reply:
To detect QR codes, we should avoid tiling, because it might crop the QR code (thinking in the cases we are close enough, to actually get accurate measurements). Instead, we should just undistort the image without tiling, and this is giving good results, the qr detector works well.